### DISCUSSION TITLE
Color theme

### DISCUSSION THREAD

**Discussion by Kaaveh** [Kaaveh/ComposeNews#124] [General]
Hi @mhmd-android
Out of curiosity, where you get these color pallet? 

![Screenshot 2023-10-05 at 11 31 51 AM](https://github.com/Kaaveh/ComposeNews/assets/39463024/d8ec2116-3ee4-45a6-b213-5d3d2e845d1c)

**Comment by Coding-Meet** [comment]
I think this website use it

https://material-foundation.github.io/material-theme-builder/
